std_msgs/Header header
int64[] object_ids
int64[] object_classes
float64[] wrist_extensions
float64[] time_to_hit
float64[] object_widths
float64[] object_heights
float64[] lifts
float64[] similarity